monkey/tests/monkey_island/cc/services/test_post_breach_files.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import pytest
+
+from monkey_island.cc.services.post_breach_files import PostBreachFilesService
+
+
+def raise_(ex):
+    raise ex
+
+
+@pytest.fixture(autouse=True)
+def custom_pba_directory(tmpdir):
+    PostBreachFilesService.initialize(tmpdir)
+
+
+def create_custom_pba_file(filename):
+    assert os.path.isdir(PostBreachFilesService.get_custom_pba_directory())
+
+    file_path = os.path.join(PostBreachFilesService.get_custom_pba_directory(), filename)
+    open(file_path, "a").close()
+
+
+def test_remove_pba_files():
+    create_custom_pba_file("linux_file")
+    create_custom_pba_file("windows_file")
+
+    custom_pda_dir_contents = os.listdir(PostBreachFilesService.get_custom_pba_directory())
+    assert len(custom_pda_dir_contents) == 2
```

Comment:
This tests `create_custom_pba_file` which is defined in the unit test. Maybe it would be better to do assertions of file created in the `create_custom_pba_file` method itself? Not a strict requirement though, you can leave as is.

Reply:
Yeah, that assert is there to make sure the test doesn't fail silently. From the test's perspective, it says:

- make sure some files are in the dir
- call `remove_PBA_files()`
- make sure the dir is emtpy

There may be a better implementation, but I don't think that moving the assert to `create_custom_pba_file()` is it. I think I can add some utility methods to make the intent clearer.